react-ui/src/components/SignUpForm.js

```diff
@@ -68,22 +73,47 @@ render() {
                 </label>
                 <br></br>
                 <br></br>
-                <label for="role" className ="Title-Style2"> Select Role: </label>
-                <select name="role" className ="InputFields2" id="type">
+                <label for="role" className ="Title-Style2"> Select User Role: </label>
+                <select name="role" className ="InputFields3" id="type">
                   <option value="student">Student</option>
+                  <option value="worker1">Worker</option>
+                </select>
+                <br></br>
+                <br></br>
+                <label for="worker_type" className ="Title-Style2"> Select Worker Type (or N/A otherwise): </label>
+                <br></br>
+                <select name="worker_type" className ="InputFields2" id="type">
+                  <option value="student">Social Worker</option>
                   <option value="worker1">Guidance Counselor</option>
-                  <option value="worker2">Master of Social Work</option>
-                  <option value="worker3">Psychologist</option>
                   <option value="other">Other</option>
+                  <option value="other">N/A</option>
                 </select>
                 <br></br>
-                {/* <br></br> */}
                   <label>
                     <input 
                       className ="InputFields" 
                       type="text" 
-                      name="optional_role"
-                      placeholder= "Enter Role, If 'Other' Selected" 
+                      name="optional_type"
+                      placeholder= "Enter Worker Type, If 'Other'" 
+                      onChange={this.handleFormChange} />
+                    </label>
+                  <br></br>
+                  <br></br>
+                <label for="specialization" className ="Title-Style2"> Select Worker Specialization (or N/A): </label>
+                <br></br>
+                <select name="specialization" className ="InputFields2" id="type">
+                  <option value="student">Masters in Social Work</option>
```

Comment:
Verified these dropdown values with the invision designs - please let me know if any values are missing/need to be added!